packages/scss/src/components/verticalNavigation/component.scss

```diff
@@ -0,0 +1,42 @@
+@use '@lucca-front/scss/src/commons/utils/a11y';
+@use '@lucca-front/scss/src/commons/utils/reset';
+
+@mixin component($atRoot: 'without: rule') {
+
+	@at-root ($atRoot) {
+		.verticalNavigation-list {
+			@include reset.list;
+			background-color: var(--colors-white-color);
+		}
+
+		.verticalNavigation-list-item-link {
+			@include reset.button;
+			display: flex;
+			padding: var(--spacings-XS) var(--spacings-S);
+			border-radius: var(--commons-borderRadius-M);
+			color: var(--components-verticalNavigation-link-color);
+			text-decoration: none;
+			background-color: var(--components-verticalNavigation-link-background);
+			transition: background-color var(--commons-animations-durations-fast) ease;
+			font-weight: 600;
+
+			.lucca-icon {
+				margin-left: auto;
+				font-size: var(--sizes-M-lineHeight);
+			}
+
+			&:hover {
+				--components-verticalNavigation-link-background: var(--palettes-grey-50);
+				color: var(--components-verticalNavigation-link-color);
```

Comment:
Probably redundant. The color is already defined l17.

Reply:
We need it to override our `a:hover` style